NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
Ok, you can ignore the commentary from the peanut gallery over here.  There was a piece to the SGA that wasn't clicking for me, someone pointed it out.  This change is fine.